docs/platforms/react-native/sourcemaps/uploading/codepush.mdx

```diff
@@ -26,25 +26,26 @@ export default codePush(Sentry.wrap(App));
 To ensure Sentry can symbolicate events from your CodePush releases, you need to generate and upload the necessary assets. When creating a CodePush release, include the `--sourcemap-output-dir` flag to generate source maps. This allows you to upload these files to Sentry in the next step.
 
 ```bash {tabTitle:JSC}
-appcenter codepush release-react \
-  --app "${APP_NAME}" \
-  --deployment-name "${DEPLOYMENT_NAME}" \
-  --output-dir ./build \
-  --sourcemap-output-dir ./build
+code-push-standalone release-react \
+  "${APP_NAME}" \
+  "${PLATFORM}" \
+  --deploymentName "${DEPLOYMENT_NAME}" --useHermes \
```

Comment:
`m`: is the `--useHermes` param here intentional? it seems to [enforce using Hermes compiler](https://github.com/microsoft/code-push-server/blob/main/cli/README.md#use-hermes-parameter)?